0.9.0 can't build for Windows

```
   Compiling jokeyrhyme-dotfiles v0.9.0
error[E0433]: failed to resolve. Use of undeclared type or module `std`
 --> src\tasks\windows.rs:9:5
  |
9 |     std::fs::create_dir_all(&bin).expect(&format!(
  |     ^^^ Use of undeclared type or module `std`

error[E0425]: cannot find value `parent` in this scope
  --> src\tasks\windows.rs:11:10
   |
11 |         &parent.display()
   |          ^^^^^^ not found in this scope

error[E0308]: mismatched types
  --> src\utils\fs.rs:56:5
   |
55 | pub fn set_executable(target: &Path) -> std::io::Result<()> {
   |                                         ------------------- expected `std::result::Result<(), std::io::Error>` because of return type
56 |     Ok
   |     ^^ expected enum `std::result::Result`, found fn item
   |
   = note: expected type `std::result::Result<(), std::io::Error>`
              found type `fn(_) -> std::result::Result<_, _> {std::result::Result<_, _>::Ok}`

error: aborting due to 3 previous errors
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+* Windows: fix the build again :S
+
 ## [0.9.0] - 2018-04-19
 
 ### Added

--- a/src/tasks/windows.rs
+++ b/src/tasks/windows.rs
@@ -1,3 +1,4 @@
+use std;
 use std::path::Path;
 
 use utils;
@@ -8,7 +9,7 @@ pub fn sync() {
     let bin = utils::env::home_dir().join(Path::new(".local/bin"));
     std::fs::create_dir_all(&bin).expect(&format!(
         "unable to create directories {}",
-        &parent.display()
+        &bin.display()
     ).as_str());
     println!("{:?}", &bin);
 

--- a/src/utils/fs.rs
+++ b/src/utils/fs.rs
@@ -53,7 +53,7 @@ pub fn set_executable(target: &Path) -> std::io::Result<()> {
 
 #[cfg(not(unix))]
 pub fn set_executable(target: &Path) -> std::io::Result<()> {
-    Ok
+    Ok(())
 }
 
 pub fn symbolic_link_if_exists(src: &Path, dest: &Path) {

